packages/story-editor/src/components/canvas/karma/quickActions.karma.js

```diff
@@ -56,6 +56,60 @@ describe('Quick Actions integration', () => {
 
       expect(fixture.screen.queryByTestId('quick-actions-menu')).toBeNull();
     });
+
+    it('quick menu should not be visible if no quick actions are present', async () => {
+      // when two different elements types are selected, there may not be any quick actions to show
+      // in that case, we shouldn't be rendering the context menu at all
+      const insertElement = await fixture.renderHook(() => useInsertElement());
+      const shapeElement = await fixture.act(() =>
+        insertElement('shape', {
+          backgroundColor: {
+            color: {
+              r: 203,
+              g: 103,
+              b: 103,
+            },
+          },
+          type: 'shape',
+          x: 48,
+          y: 0,
+          width: 148,
+          height: 137,
+          scale: 100,
+          focalX: 50,
+          focalY: 50,
+          mask: {
+            type: 'heart',
+          },
+          id: 'cb89750a-3ffd-4876-8ed9-d715c553e05b',
+          link: null,
+        })
+      );
+
+      await fixture.act(() =>
+        insertElement('text', {
+          font: TEXT_ELEMENT_DEFAULT_FONT,
+          content: 'Hello world!',
+          x: 10,
+          y: 20,
+          width: 400,
+        })
+      );
+
+      await fixture.editor.canvas.framesLayer.waitFocusedWithin();
+
+      expect(
+        fixture.screen.queryByTestId('Element quick actions')
+      ).not.toBeNull();
+
+      await fixture.events.keyboard.down('Shift');
+      await clickOnTarget(
+        fixture.editor.canvas.framesLayer.frame(shapeElement.id).node
+      );
+      await fixture.events.keyboard.up('Shift');
```

Comment:
💭  Would be good to verify that the quick action menu is visible prior to selecting the second element.

Reply:
I am checking for it. Lines 101-103

Reply:
Oh great! Sorry, I missed it